ui/components/Pagination.tsx

```diff
@@ -0,0 +1,113 @@
+import { FormControl, MenuItem, Select } from "@material-ui/core";
+import * as React from "react";
+import styled from "styled-components";
+import Button from "./Button";
+import Flex from "./Flex";
+import Icon, { IconType } from "./Icon";
+import Spacer from "./Spacer";
+import Text from "./Text";
+
+export interface Props {
+  /** CSS MUI Overrides or other styling. */
+  className?: string;
+  /** func for forward one page button */
+  forward: () => void;
+  /** func for skip to last page button */
+  skipForward: () => void;
+  /** func for back one page button */
+  back: () => void;
+  /** func for skip to start button */
+  skipBack: () => void;
```

Comment:
Typically, you would prefix an event handler with `on` to indicate its a handler: `onSkipBack` etc